docs/make_example.jl

```diff
@@ -52,8 +54,9 @@ end
 
 example_pages = [
                  #"Internal wave"                    => "generated/internal_wave.md",
-                 "Geostrophic adjustment"            => "generated/geostrophic_adjustment.md"
-                ]
+                 #"Geostrophic adjustment"            => "generated/geostrophic_adjustment.md",
+                 "Bickley jet shallow water model"    => "generated/Bickley_jet_shallow_water.md"
```

Comment:
```suggestion
                 "Bickley jet in Shallow Water model"    => "generated/Bickley_jet_shallow_water.md"
```

Reply:
Better

Reply:
I meant to suggest the change in the `make.jl` file as well!
(Actually it's there that it really matters... :))